src/main/resources/openapi/sqs/approved-postal-vote-application-sqs-messaging.yaml

```diff
@@ -0,0 +1,146 @@
+    PostalVoteDetails:
+      type: object
+      properties:
+        ballotAddress:
+          $ref: '../sqs/Models/Address.yaml'
+        ballotAddressReason:
+          type: string
+          description: ' (required only if “ballot” address given) The reason the applicant gave for using a different address for the ballot paper (max 500 utf-8 chars)'
+          maxLength: 500
+        postalVoteUntilFurtherNotice:
+          type: boolean
+          description: Mutually exclusive postal voting limits. True if the elector wishes to vote by post until further notice.
+        postalVoteForSingleDate:
+          type: string
+          format: date
+          description: |
+            If postal application is for a single dates.
+            In ISO8601 date format '2022-08-02'
+        postalVoteStartDate:
+          type: string
+          format: date
+          description: |
+            If set, the elector wishes to vote by post for the period defined by the two dates. If present, then postalVoteEndDate must also be present
+            In ISO8601 date format '2022-08-02'
+        postalVoteEndDate:
+          type: string
+          format: date
+          description: |
+            If set, the elector wishes to vote by post for the period defined by the two dates. If present, then postalVoteStartDate must also be present
+            In ISO8601 date format '2022-08-02'
+
+    ApprovedPostalVoteApplicationMessage:
+      title: ApprovedPostalVoteApplicationMessage
+      x-api-owner: EMS
+      x-api-consumer: EROP
+      type: object
+      x-examples:
+        Example 1:
+          approvalDetails:
+            id: 502cf250036469154b4f85fc
+            createdAt: 2022-08-01T14:14:55Z
+            gssCode: T12345678
+            source: web
+            authorisedAt: 2022-08-02T10:10:10Z
+            authorisingStaffId: jsmith99@somecouncil.gov.uk
+          applicantDetails:
+            firstName: First
+            middleNames: Middle
+            lastName: Last
+            dob: '1995-03-02'
+            registeredAddress:
+              property: Aviation House
+              street: 125 Kingsway
+              locality: Camden
+              town: London
+              area: Greater London
+              postcode: SW112DR
+              uprn: 098765
+            phone: 07799779977
+            email: john@example.com
+            refNum: V3HQ9LKI0W
+            ip: 1.2.3.4
+            lang: en
+            emsElectorId: e87cbaea-0deb-4058-95c6-8240d426f5e1
+          postalVoteDetails:
+            ballotAddress:
+              property: Another House
+              street: 234 Kingsway
+              locality: Camden
+              town: London
+              area: Greater London
+              postcode: SW112DR
+              uprn: '098766'
+            ballotAddressReason: I am away from home for work
+            signature: LzlqLzRBQ...
+            postalVoteUntilFurtherNotice: true
+          signature: LzlqLzRBQ...
```

Comment:
Duplicate signature field - prob the one in postalVoteDetails can be removed?

Reply:
Removed